api/src/opentrons/protocol_engine/resources/deck_data_provider.py

```diff
@@ -49,8 +49,8 @@ async def get_deck_fixed_labware(
 
         for fixture in deck_definition["locations"]["fixtures"]:
             labware_id = fixture["id"]
-            load_name = fixture.get("labware")  # type: ignore[misc]
-            slot = fixture.get("slot")  # type: ignore[misc]
+            load_name = cast(Optional[str], fixture.get("labware", None))
+            slot = cast(Optional[str], fixture.get("slot", None))
```

Comment:
`None`'s are redundant

```suggestion
            load_name = cast(Optional[str], fixture.get("labware"))
            slot = cast(Optional[str], fixture.get("slot"))
```